src/pages/TeamSettings/KV.vue

```diff
@@ -218,28 +207,12 @@ export default {
           return
         }
       }
-      let value = this.KvValueInput
```

Comment:
Tested KV store using Netlify and see a similar error to the secrets - need error handling (that is in the original version but looks like it's not working here) and to be able to handle a string that is incorrect json (since the expectation is that both strings and json could be added here): 





<img width="621" alt="Screen Shot 2021-11-13 at 11 06 12 AM" src="https://user-images.githubusercontent.com/40272060/141651169-3130f59f-8ec2-426d-9a97-6fef99538b01.png">


<img width="1227" alt="Screen Shot 2021-11-13 at 10 55 02 AM" src="https://user-images.githubusercontent.com/40272060/141651174-c6475abb-cc19-41b3-8769-6612f8b3879e.png">

<img width="579" alt="Screen Shot 2021-11-13 at 11 08 10 AM" src="https://user-images.githubusercontent.com/40272060/141651152-736430e0-5ebe-4a1b-8732-f723a2f51514.png">

Reply:
I believe these concerns should be addressed with latest commits, thanks for the great suggestions